docs/topics/faq.md

```diff
@@ -72,14 +72,11 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
 
 ### What IDEs support Kotlin?
 
-Kotlin is supported by all major Java IDEs including [IntelliJ IDEA](https://www.jetbrains.com/idea/download/),
-[Android Studio](https://developer.android.com/kotlin/get-started), and [Eclipse](https://www.eclipse.org/downloads/). In addition, a [command line compiler](command-line.md) 
-is available and provides straightforward support for compiling and running applications.
-
-> Kotlin Plugin for Eclipse has limited support.  
-> Use [other IDEs for Kotlin development](kotlin-ide.md) to get full support of new Kotlin features.
->
-{type="warning"}
+Kotlin has full out-of-the-box support in [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) and
+[Android Studio](https://developer.android.com/kotlin/get-started) by official Kotlin plugin developed by JetBrains.
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins in marketplaces.
```

Comment:
I'd drop "in marketplaces" here. Where are those community-supported plugins is not relevant and might vary over time.